tests/integration/activities.test.js

```diff
@@ -129,6 +129,39 @@ describe('ACTIVITY ROUTES - PUT /activity/{_id}', () => {
       expect(activityUpdated).toEqual(expect.objectContaining({ _id: activityId, name: 'rigoler' }));
     });
 
+    it('should update cards', async () => {
+      const payload = {
+        cards: [
+          activitiesList[0].cards[1],
+          activitiesList[0].cards[0],
```

Comment:
ça me trigger beaucoup trop que activitiesList[0].cards[0] soit après activitiesList[0].cards[1]. 🙀

Reply:
c'est justement pour tester le drag and drop => je change l'ordre des cartes ;)